lms/templates/student_account/login.underscore

```diff
@@ -1,14 +1,24 @@
 <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
 </div>
 
-<% if ( context.createAccountOption !== false ) { %>
+<% if ( context.createAccountOption !== false && context.enterpriseName == "") { %>
```

Comment:
I do not think we need this check, I think [this](https://github.com/edx/edx-platform/pull/16691/files#diff-7960baf09ff3f493a2b3fedefa7aad05R164) change in https://github.com/edx/edx-platform/pull/16691 will generate required behavior